third_party/terraform/utils/field_helpers.go

```diff
@@ -231,6 +231,17 @@ func getProjectFromSchema(projectSchemaField string, d TerraformResourceData, co
 	return "", fmt.Errorf("%s: required field is not set", projectSchemaField)
 }
 
+func getBillingProjectFromSchema(billingProjectSchemaField string, d TerraformResourceData, config *Config) (string, error) {
```

Comment:
Can you update https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/pre_create/cloud_asset_feed.go.erb to use this function?